app/uk/gov/hmrc/bindingtariffclassification/common/Logging.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.common
+
+import org.slf4j.{Logger, LoggerFactory}
+
+trait Logging {
+  val logger: Logger = LoggerFactory.getLogger(this.getClass)
+}
```

Comment:
I think we can use `play.api.Logging` rather than defining this ourselves

Reply:
Unfortunately this is only in play 2.7 +